docs/examples/Rocket_Car.rst

```diff
@@ -0,0 +1,1585 @@
+##
+
+.. raw:: html
+
+   <center>
+
+:math:`\mathbf{\text{Modeling of a Variable-Mass Nonholonomic Gyrostatic Rocket Car Using Extended Kane's Equations}}`
+
+.. raw:: html
+
+   </center>
+
+#####
+
+.. raw:: html
+
+   <center>
+
+:math:`\text{Ge, Z.M. and Cheng, Y.H., 1982. Extended Kane’s equations for nonholonomic variable mass system.}`
+
+.. raw:: html
+
+   </center>
+
+#####
+
+.. raw:: html
+
+   <center>
+
+:math:`\text{Figure 1: Idealized model of a jet racing car that is propelled by a rocket engine at point P, the rocket engine being treated as a variable mass particle at P}`
+
+.. raw:: html
+
+   </center>
+
+#####
+
+.. raw:: html
+
+   <center>
+
+:math:`\text{Here}, \{a_x: g_3, a_y: g_1, a_z: g_2\}`
+
+.. raw:: html
+
+   </center>
+
+#####
+
+.. raw:: html
+
+   <center>
+
+:math:`\text{Leimanis, E., 2013. The general problem of the motion of coupled rigid bodies about a fixed point (Vol. 7). Springer Science & Business Media. (Page 207)}`
+
+.. raw:: html
+
+   </center>
+
+.. code:: ipython3
```

Comment:
All code input should use the ``.. jupyter-execute::`` directive. See the other examples for usage: https://raw.githubusercontent.com/pydy/pydy/master/docs/examples/carvallo-whipple.rst